documentation/simplified-grid/grid.css

```diff
@@ -0,0 +1,74 @@
+@import "/documentation/Styles/general.css";
+
+.container {
+    margin-top: 6.5rem;
+    padding: 1rem;
+}
+
+.grid-rows {
+    display: grid;
+    justify-items: center;
+    text-align: center;
```

Comment:
You can use the shorthand `place-items: center` to vertically and horizontally center items

Reply:
Thanx, I Will apply that.